### PR TITLE
Completed project on portfolio with html and css

### PR DESCRIPTION
**Pull request by Mekuseo**
This is the second part of the project to complete my portfolio using HTML and CSS. 
This part of the project involves using flexbox and grid to build the other part of the portfolio

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,38 +1,298 @@
...
-            <img src="./images/mobile images.png" alt="mobile-link">
-        </div>
-        <nav class="header2">
-            <h3>Welcome</h3>
-            <img class='menu-button' src="./images/Enabled.png" alt="mobile-menu">
-        </nav>
+      <div class="header1">
+        <h3>9:41</h3>
+        <img src="./images/mobile images.png" alt="mobile-link" />
+      </div>
```
Comment (pinkmoon25) [inline index.html]: [OPTIONAL] You actually do not need to add this div to your portfolio as it is not part of the template design but just to mimic the phone screen, your header should have a logo inclosed in `<a>` tag and a `<nav>` bar for the hamburger button only.

File: index.html [hunk 2]
```diff
@@ -1,38 +1,298 @@
...
+            <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+            <div class="card-tags">
+              <p>Ruby on Rails</p>
+              <p>css</p>
+              <p>JavaScript</p>
+              <p>html</p>
+            </div>
+            <button class="card-button">See Project</button>
+          </div>
+        </div>
```
Comment (pinkmoon25) [inline index.html]: - [x] As mentioned in the project requirements **_"Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card."_**   please use flex for layout of each individual card, we need you to be flex ninja :ninja:

File: index.html [hunk 3]
```diff
@@ -1,38 +1,298 @@
...
+              alt=""
+              class="card-image"
+            />
+            <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+            <div class="card-tags">
+              <p>Ruby on Rails</p>
+              <p>css</p>
+              <p>JavaScript</p>
+              <p>html</p>
+            </div>
```
Comment (pinkmoon25) [inline index.html]: - [x] As mentioned in the project requirements _**"Please use `<ul>` and `<li>` for categories in "Work" section. Html, Css, and Javascript should be list items."**_ , It is semantically more appropriate to use `<ul>` tag for listing technologies and use `<li>` tag for each individual technology, Please make your card-tags `<ul> `with `<li>` elements inside.

File: index.html [hunk 4]
```diff
@@ -1,38 +1,298 @@
...
+                <h4 class="footer-tags">codekit</h4>
+                <h4 class="footer-tags">Github</h4>
+                <h4 class="footer-tags">Codepen</h4>
+              </div>
+              <div class="footer-list">
+                <h4 class="footer-tags">Gitlab</h4>
+                <h4 class="footer-tags">Terminal</h4>
+              </div>
+            </div>
+          </div>
```
Comment (pinkmoon25) [inline index.html]: - [x] Please :pray:  remove all this from a `<footer>` tag and use a `<section>` tag to group all this, as your About myself is not supposed to be inside a <footer>`  but a `<section>` tag, the `<footer>` should contain only the contact form and the text that says 'i am always interested...'  
- [x] Also use flex box for the layout of about myself section

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @Mekuseo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
linters are passing :heavy_check_mark: 
project looks professional :heavy_check_mark: 
follows the figma design :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Mekuseo) [reply index.html]: Done. Thank you for your corrections

File: index.html (hunk 2 above)
Comment (Mekuseo) [reply index.html]: Corrected. Thank you for your review

File: index.html (hunk 3 above)
Comment (Mekuseo) [reply index.html]: corrected. Thank you

File: index.html (hunk 4 above)
Comment (Mekuseo) [reply index.html]: I have removed the footer and replaced it with section as suggested though I do not understand your other directions <but a tag, the >. I do not understand that.

about myseflf section is already in flex box as you have suggested

File: index.html [hunk 5]
```diff
@@ -1,38 +1,320 @@
...
+      </section>
+    </main>
+      <section class="about-us">
+        <h1 class="about-heading">About Me</h1>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, do not hestiate to contact me.
+        </p>
+      </section>
```
Comment (christianbiring1) [inline index.html]: - [x] As suggested by the previous reviewer, Kindly use the `section tag` is supposed to grab from the About up to the end of the skills part. Make sure you remove all the other `section` tag and replace them even with some generic `div`

File: index.html [hunk 6]
```diff
@@ -1,38 +1,320 @@
...
+                  <h4 class="footer-tags">Ruby</h4>
+                </div>
+                <div class="footer-list">
+                  <h4 class="footer-tags">Html</h4>
+                  <h4 class="footer-tags">CSS</h4>
+                </div>
+              </div>
+            </div>
+          </div>
+        </section>
```
Comment (christianbiring1) [inline index.html]: - [x] Section doesn't have any semantic meaning here. Kindly replace that. 
You can check [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) when to use a `section` tag
`HINT: You can use a generic div if you want here.`

File: index.html [hunk 7]
```diff
@@ -1,38 +1,320 @@
...
+                <h4 class="footer-tags">Ruby on Rails</h4>
+              </div>
+              <div class="footer-list">
+                <h4 class="footer-tags">RSpec</h4>
+                <h4 class="footer-tags">Capybara</h4>
+                <h4 class="footer-tags">Selenium</h4>
+              </div>
+            </div>
+          </div>
+        </section>
```
Comment (christianbiring1) [inline index.html]: - [x] Section doesn't have any semantic meaning here. Kindly replace that. 
You can check [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) when to use a `section` tag
`HINT: You can use a generic div if you want here.`

File: index.html [hunk 8]
```diff
@@ -1,38 +1,320 @@
...
+                <h4 class="footer-tags">Github</h4>
+                <h4 class="footer-tags">Codepen</h4>
+              </div>
+              <div class="footer-list">
+                <h4 class="footer-tags">Gitlab</h4>
+                <h4 class="footer-tags">Terminal</h4>
+              </div>
+            </div>
+          </div>
+        </section>
```
Comment (christianbiring1) [inline index.html]: - [x] Section doesn't have any semantic meaning here. Kindly replace that. 
You can check [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) when to use a `section` tag
`HINT: You can use a generic div if you want here.`

File: index.html [hunk 9]
```diff
@@ -1,38 +1,320 @@
...
+        <h1 class="footer-header">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h1>
```
Comment (christianbiring1) [inline index.html]: [OPTIONAL]
- [x] Also you may consider using the same font-size as the template in Figma 
Your design :point_down: 
image


Figma design :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @Mekuseo , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
# STATUS /;REQUEST CHANGES :recycle: 
soclose


 ## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] Good commit message :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 5 above)
Comment (Mekuseo) [reply index.html]: All section tags in About us section is removed and only one section tag covering frim About right up the end of skills

File: index.html (hunk 6 above)
Comment (Mekuseo) [reply index.html]: Corrected. Section tag removed and replaced with div.

File: index.html (hunk 7 above)
Comment (Mekuseo) [reply index.html]: Corrected. Section tag removed and replaced with div.

File: index.html (hunk 8 above)
Comment (Mekuseo) [reply index.html]: Corrected. Section tag removed and replaced with div.

File: index.html (hunk 9 above)
Comment (Mekuseo) [reply index.html]: Corrections done

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @Mekuseo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the set up the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**